crane-cloud/frontend
# Add dashboard for Crane Cloud Status Monitoring

# Description

This Pull Request adds a provision to view the high level status overview of services offered and used by Crane Cloud.

## Type of change


## Trello Ticket ID

Please add a link to the Trello ticket for the task.

## How Can This Been Tested?

Pull and run this branch locally and checkout the `http://localhost:3000/status` to see the added interface.

## Checklist:


# Screenshots

![image](https://user-images.githubusercontent.com/63339234/177698556-be2e0a0e-aa9f-4d9c-bd8f-e824283bea21.png)
